test/WorkflowCore.Tests.YmalDefinition/YmalDefinitionTest.cs

```diff
@@ -0,0 +1,46 @@
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using WorkflowCore.Interface;
+using WorkflowCore.Testing;
+using Xunit;
+
+namespace WorkflowCore.Tests.YmalDefinition
+{
+    public class YmalDefinitionTest:YamlWorkflowTest
```

Comment:
The changes you made are not YAML specific, they apply to the definition loader, regardless of serialization format.  I think we should add unit tests [https://github.com/danielgerlag/workflow-core/blob/master/test/WorkflowCore.UnitTests/Services/DefinitionStorage/DefinitionLoaderTests.cs](here). This particular test is more of an integration test, which we could add to the integration tests project.  I think the unit tests are more important in this case.

Reply:
Thanks,good suggesttion, I have  added to  unit tests.